packages/firestore/test/integration/api/query.test.ts

```diff
@@ -1614,6 +1615,34 @@ apiDescribe('Queries', (persistence: boolean) => {
       });
     });
   });
+
+  // eslint-disable-next-line no-restricted-properties
+  (persistence ? it.skip : it)(
+    'can raise expected snapshot when resume query after deleting docs',
```

Comment:
Consider renaming this test to something like "resuming a query should remove deleted documents indicated by existence filter"